docker-compose.yml

```diff
@@ -112,7 +112,7 @@ services:
             - "2182:2181"
 
     kafka-no-auth:
-        image: wurstmeister/kafka:latest
+        image: wurstmeister/kafka:2.11-1.1.0
```

Comment:
Why not 2.3.1?

Reply:
@afalko I pushed a new commit to update to 2.3.0 because wurstmeister/kafka image hasn't been updated to 2.3.1 yet

Reply:
Thanks! That's close enough for now :)